dev/AppNotifications/ShellLocalization.cpp

```diff
@@ -188,9 +188,8 @@ inline std::path RetrieveLocalFolderPath()
     THROW_IF_FAILED(SHGetKnownFolderPath(FOLDERID_LocalAppData, 0 /* flags */, nullptr /* access token handle */, &localAppDataPath));
 
     // path: C:\Users\<currentUser>\AppData\Local\Microsoft\WindowsAppSDK
-    std::path localFolderPath{ std::wstring(localAppDataPath.get()) +
-        Microsoft::Windows::AppNotifications::ShellLocalization::c_localMicrosoftFolder +
-        Microsoft::Windows::AppNotifications::ShellLocalization::c_localWindowsAppSDKFolder };
+    std::path localFolderPath{ localAppDataPath.get() };
+    localFolderPath /= Microsoft::Windows::AppNotifications::ShellLocalization::c_localWindowsAppSDKFolder;
```

Comment:
Why multiple lines?

Reply:
In this case, we were creating an unnecessary std::wstring. Concatenation can't happen with a raw string to the left.

Reply:
Got it. Yes, unecessary std::wstring conversion is bad. /= is only overloaded for std::path, not raw strings.